app/locale/zh-HANS.coffee

```diff
@@ -1084,9 +1084,9 @@ module.exports = nativeDescription: "简体中文", englishDescription: "Chinese
 #    stephanie_title: "Customer Support Specialist"
 #    rob_title: "Sales Development Representative"
 #    shubhangi_title: "Software Engineer"
-    bryukh_title: "游戏设计师"
-    retrostyle_title: "插画师"
-    retrostyle_blurb: "复古风格的游戏"
+#    bryukh_title: "游戏设计师"
+#    retrostyle_title: "插画师"
+#    retrostyle_blurb: "复古风格的游戏"
     bryukh_blurb: "构建拼图"
```

Comment:
Is this blurb going to be used?

Reply:
Not now. We are going to just keep English title now.